prov/usnic/src/usdf_dgram.c

```diff
@@ -167,40 +169,40 @@ usdf_dgram_senddata(struct fid_ep *fep, const void *buf, size_t len,
 }
 
 static ssize_t
-_usdf_dgram_send_iov_copy(struct usdf_ep *ep, struct usd_dest *dest,
-		const struct iovec *iov, size_t count, void *context)
+_usdf_dgram_send_iov_copy(struct usdf_ep *ep, const struct usd_dest *dest,
+		const struct iovec *iov, size_t count, void *context,
+		uint8_t cq_entry)
 {
-	struct usd_wq *wq;
- 	struct usd_qp_impl *qp;
+	struct usd_wq_post_info *info;
 	struct usd_udp_hdr *hdr;
+	struct usd_qp_impl *qp;
+	struct usd_wq *wq;
+
 	uint32_t last_post;
-	struct usd_wq_post_info *info;
 	uint8_t *copybuf;
-	size_t len;
-	unsigned i;
+	size_t len = sizeof(struct usd_udp_hdr);
 
 	qp = to_qpi(ep->e.dg.ep_qp);
 	wq = &qp->uq_wq;
-	copybuf = wq->uwq_copybuf +
-			wq->uwq_post_index * USD_SEND_MAX_COPY;
 
-	hdr = (struct usd_udp_hdr *)copybuf;
+	copybuf = wq->uwq_copybuf + (wq->uwq_post_index * USD_SEND_MAX_COPY);
+
+	hdr = (struct usd_udp_hdr *) copybuf;
+
 	memcpy(hdr, &dest->ds_dest.ds_udp.u_hdr, sizeof(*hdr));
-	hdr->uh_udp.source =
-		qp->uq_attrs.uqa_local_addr.ul_addr.ul_udp.u_addr.sin_port;
 
-	len = sizeof(*hdr);
-	for (i = 0; i < count; i++) {
+	for (size_t i = 0; i < count; i++) {
```

Comment:
Was there a good reason to convert the existing `unsigned i` declaration at the top of the file into this `size_t i` declaration in the `for` loop?